keyboards/xiudi/xd60/info.json

```
@@ -369,6 +369,76 @@
                 {"matrix": [4, 13], "x": 13.75, "y": 4, "w": 1.25}
             ]
         },
+        "LAYOUT_60_tsangan": {
```

```suggestion
        "LAYOUT_60_ansi_tsangan_split_bs": {
```
This board also supports `60_ansi_tsangan_split_bs_rshift` (currently the community layout for this is named `60_tsangan_hhkb`), `60_iso_tsangan_split_bs` (not a community layout but matches this), and `60_iso_tsangan_split_bs_rshift`.

---

@fauxpark 
I've covered off supported community layouts for this board in #21832